mhkit/tests/Utils_TestGenUtils.m

```diff
@@ -71,6 +73,142 @@ function test_magnitude_phase(testCase)
             assert(all(phase1 == theta))
             assert(all(phase2 == phi))
         end
+        function test_read_nc_file_group(testCase)
+            % Check MATLAB version
+            if isMATLABReleaseOlderThan("R2021b")
```

Comment:
You can remove this version check. We will set version requirements at the MHKiT package level. We do not want to have a "return" on this for skipping the test because it may cause issues to be left uncaught.